classes/models/FrmOnSubmitAction.php

```
@@ -45,6 +45,8 @@ public static function get_name() {
 	/**
 	 * @param object $instance
 	 * @param array  $args
+	 *
+	 * @return void
 	 */
 	public function form( $instance, $args = array() ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$instance'. (undefined)

(UnusedFormalParameter)